roles/oVirt.datacenter-cleanup/tasks/find_hosted_engine_details.yml

```diff
@@ -0,0 +1,40 @@
+---
+- name: Find existing VMs
+  ovirt_vms_facts:
+    auth: "{{ ovirt_auth }}"
+    pattern: "datacenter={{ data_center_name }}"
+
+- name: Find HE info if HE
+  set_fact:
+    hosted_engine: "{{ ovirt_vms | selectattr(search_key,'equalto',search_val) | list | count > 0 }}"
+  vars:
+    search_key: origin
+    search_val: managed_hosted_engine
+
+- name: Print information about VM running the engine
```

Comment:
This actually doesn't print information about VM running the engine cause you are getting this info later on, so please move it bellow and print the proper name of vm running HE if you would like this info in debug

Reply:
fixed